src/cli_options.rs

```diff
@@ -0,0 +1,344 @@
+use nu_cli::report_error;
+use nu_engine::{get_full_help, CallExt};
+use nu_parser::{escape_quote_string, parse};
+use nu_protocol::ast::{Call, Expr, Expression, PipelineElement};
+use nu_protocol::engine::{Command, EngineState, Stack, StateWorkingSet};
+use nu_protocol::{
+    Category, Example, IntoPipelineData, PipelineData, ShellError, Signature, Spanned, SyntaxShape,
+    Value,
+};
+use std::io::Write;
+
+#[derive(Clone, Debug)]
+pub struct CliOptions {
+    pub conn_string: Option<String>,
+    pub username: Option<String>,
+    pub password: bool,
+    pub cluster: Option<String>,
```

Comment:
so, I guess these stay the same in the move to database from cluster?

Reply:
These are just struct members used internally. Parsing into them happens below. You're right that cluster need renaming to database in the parsing too though.

Reply:
Filed #229